docs-user/async-posix-signal-control.md

```diff
@@ -0,0 +1,102 @@
+
+# Async POSIX Signal Control
```

Comment:
Now that we moved this feature to our user docs, I'm wondering if we should change the title to describe why it's used for to make it clearer.

For example something like "Profiling with Async POSIX Signals" would make it clearer that POSIX signals are used for profiling. It would also be more aligned with the other items in the sidebar.